@uportal/esco-content-menu-lit/README.md

```diff
@@ -0,0 +1,643 @@
+- `background-color`: type: `String`, default: `rgba(0, 0, 0, 0)`, to apply a different background-color
+- ~~`call-after-action`: type: `Function`, default: `undefined`, a callback function to call into `portlet-card` embeding `action-favorite` after adding portlet to favorites,~~ Deprecate, use event `toggle-favorite` instead.
+- `context-api-url`: type: `String`, default: `/uPortal`, usefull to provide a different uPortal context on which to do request,
+- `favorite-api-url`: type: `String`, default: `/uPortal/api/layout`, the uri/url of the favorites api
+- `layout-api-url`: type: `String`, default: `/uPortal/api/v4-3/dlm/layout.json`, the uri/url of the layout api to request the favorite list in the other defined order (only needed to get favorite's order defined by the user)
+- `portlet-api-url`: type: `String`, default: `/uPortal/api/v4-3/dlm/portletRegistry.json`, the uri/url of the portletRegistry api to obtains user authorized portlet list
+- `user-info-api-url`: type: `String`, default: `/uPortal/api/v5-1/userinfo`, url/uri on which the api request is done to obtain user information and the jwt token
+- `portlet-card-size`: type: possible value `auto|large|medium|small|smaller|custom`, default: `auto`, define the size of `portlet-cards` component.
+- `hide-action`: type: `Boolean`, default: `false`, define to hide or not the `action-favorite` button defined into `portlet-card`
+- `show-footer-categories`: `Boolean`, default: `false`, define to display category dropdown filter near bottom of grid
+- `hide-title`: `Boolean`, default: `false`, define to remove the title area from the grid, useful when a basic grid is desired
+- `portlet-background-is-dark`: `Boolean`, default: `false`, indicate to the component that the parent background is dark and permit to the portlet-card component to show buttons like favorites in a more suitable color.
+- `debug`: type: `Boolean`, default: `false`, for the demo/debug mode to be able to run in a standalone way (disable api call)
+- `card-hover-src`: type: possible value `none|title|desc|description`, default: `none`, set the hover text for the grid cards to one of the portlet definition values `title` or `description`
+
+and additional properties to work with the parent component `content-menu`:
+
+- `parent-screen-size`: type: possible value `large|medium|small|smaller|custom`, default: `medium`, permit to indicate the breakpoint view of the parent.
+- `portlets`: type: `Array`, default: `[]`, used if the list of portlets is loaded and provided from a parent component,
+- `favorites`: type: `Array`, default: `[]`, used if the list of favorites portlets loaded and provided from a parent component,
+- `all-categories`: type: `Array`, default: `[]`, set or get the list of categories,
+
+#### Events
+
+##### Emited
+
+- `gridFavoritesUpdated`, target : `window`, details: `fname,chanId,isFavorite`, emited when a portlet is added to favorites.
```

Comment:
Can the component align on a naming convention, looks like `grid-favorites-updated` may be more appropriate?